solver/pb/attr.go

```diff
@@ -6,6 +6,7 @@ const AttrAuthHeaderSecret = "git.authheadersecret"
 const AttrAuthTokenSecret = "git.authtokensecret"
 const AttrKnownSSHHosts = "git.knownsshhosts"
 const AttrMountSSHSock = "git.mountsshsock"
+const AttrCommitHash = "git.commithash"
```

Comment:
Bikeshedding: this could be also named `AttrGitChecksum` to follow `AttrHTTPChecksum`

Reply:
@tonistiigi Which one do you prefer?

Reply:
AttrGitChecksum

Reply:
Implemented in
- https://github.com/moby/buildkit/pull/5975